sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,10 +10,6 @@
 
 - Removed `noexcept` specification from `Azure::Core::Context::IsCancelled()`.
```

Comment:
We should consider fixing up other instances of noexcept and release them all together in one GA release, rather than across multiple releases, especially if we find any others that are public, i.e. this issue:
https://github.com/Azure/azure-sdk-for-cpp/issues/3674